docs/dotnet-five.md

```diff
@@ -0,0 +1,138 @@
+
+For more information on source generators, see [Introducing C# source generators](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators) and [C# source generator samples](https://devblogs.microsoft.com/dotnet/new-c-source-generator-samples).
+
+### F# updates
+
+F# is the .NET functional programming language, and with .NET 5, developers have access to F# 5. Here are several new features of F# 5, for more information, see [Announcing F# 5](https://devblogs.microsoft.com/dotnet/announcing-net-5-0-preview-8).
+
+#### Interpolated strings
+
+Similar to interpolated string in C#, and even JavaScript - F# supports basic string interpolation.
+
+```fsharp
+let name = "David"
+let age = 36
+let message = $"{name} is {age} years old."
+```
+
+In addition to basic string interpolation, there is typed interpolation. With typed interpolation, a given type must match the format specifier.
+
+```fsharp
+let name = "David"
+let age = 36
+let message = $"%s{name} is %d{age} years old."
+```
+
+### Visual Basic updates
+
+Going forward, there is no plan to evolve Visual Basic as a language. This supports language stability and maintains compatibility between the .NET Core and .NET Framework versions of Visual Basic. Future features of .NET that require language changes may not be supported in Visual Basic.
+
+With .NET 5, Visual Basic support is extended to:
+
+- Class library
+- Console
+- Windows Forms
+- Windows Presentation Foundation (WPF)
+- Worker Service
+- ASP.NET Core Web API
+
+For more information, see [Visual Basic support planned for .NET 5](https://devblogs.microsoft.com/vbteam/visual-basic-support-planned-for-net-5-0).
+
+## What happens to Xamarin
+
+As part of the .NET unification, [Xamarin.iOS](/xamarin/ios) and [Xamarin.Android](/xamarin/android) will become part of .NET 6 as .NET for iOS and .NET for Android. Because these bindings are projections of the SDKs shipped from Apple and Google, nothing changes there, however build tooling, target framework monikers, and runtime framework monikers will be updated to match all other .NET 6 workloads. Our commitment to keeping .NET developers up to date with the latest mobile SDKs is foundational to .NET MAUI and remains firm. When .NET 6 is released, the final release of Xamarin SDKs in their current form will also release, and they'll be serviced for a year. All modern work will at that time shift to .NET 6.
+
+[Xamarin.Forms](/xamarin/get-started) will ship a new major version later this year, and continue to ship minor and service releases every 6 weeks through .NET 6 GA in November 2021. The final release of Xamarin.Forms will be serviced for a year after shipping, and all modern work will shift to .NET MAUI.
+
+### .NET MAUI
```

Comment:
How do I use .NET MAUI today? If I can't use it then it might not be worth mentioning here.

Reply:
I'm adding the links to the sample source code for it, for those who are eager to have a go of it.